source/devices/AM62L/linux/RT_Linux_Performance_Guide.rst

```diff
@@ -0,0 +1,68 @@
+======================================
+ RT-linux 11.00 Performance Guide
+======================================
+
+.. rubric::  **Read This First**
+
+**All performance numbers provided in this document are gathered using
+following Evaluation Modules unless otherwise specified.**
+
++----------------+---------------------------------------------------------------------------------------------------+
+| Name           | Description                                                                                       |
++================+===================================================================================================+
+| AM62L EVM      | AM62L Starter Kit rev E1 with ARM running at 1200MHz, LPDDR4 data rate 3200 MT/S                  |
```

Comment:
s/Starter Kit/Evaluation Module/ 

3200MT/s is a little fast. You may want to double check this line